server/app.py

```diff
@@ -144,9 +144,14 @@ def add_suffix_to_name(stop):
 
 def helper_thread():
     while True:
-        update_cache()
-        logging.debug("helper thread updated cache with predictions")
-        MetricsHandler.cache_last_updated.set(int(time.time()))
+        try:
+            update_cache()
+            logging.debug("helper thread updated cache with predictions")
+            MetricsHandler.cache_last_updated.set(int(time.time()))
+        except Exception:
+            logging.exception("Unable to update cache")
+            MetricsHandler.cache_update_errors.inc() 
```

Comment:
we might want to instead store the timestamp of when the cache had an error, lets keep this for now and see what happens in grafana after this pr is merged